daml-lf/validation/src/main/scala/com/digitalasset/daml/lf/validation/DependencyVersion.scala

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) 2020 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.lf.validation
+
+import com.daml.lf.data.Ref._
+import com.daml.lf.language.Ast._
+import com.daml.lf.transaction.VersionTimeline
+
+private[validation] object DependencyVersion {
+
+  @throws[ValidationError]
+  def checkPackage(world: World, pkgId: PackageId, pkg: Package): Unit = {
+    import VersionTimeline.Implicits._
+
+    for {
+      pkgFirstModule <- pkg.modules.values.take(1)
```

Comment:
Note that a package can have 0 modules (and you can produce such a package with the compiler) but it still has an LF version. We really need to move the version to the package to address this properly but I’m fine with leaving that to a separate PR.

Reply:
Let do that in the other PR.

Reply:
I agree, let's do it in a separate PR, but let's do it right after this one. We don't want to have this state in a release.